assignments/week_1/exercise_1_1/exercise_1_1.md

```diff
@@ -0,0 +1,163 @@
+# Exercise 1.1
+Welcome to your first exercise of RoboJackets software training! Exercises are shorter and simpler assignments where you will be walked through how to write certain code. This exercise will be testing your knowledge of basic C++ syntax as well as basic programming concepts. For this exercise we expect you to have watched or know the content of the Codecademy C++ course up through vectors.
+
+# Exercise Objective
+In this exercise you will be controlling the movements of a car like robot with 4 wheels via the command line. The robot looks like a simple car with 4 wheels, each numbered as shown below. All 4 wheel speeds are controlled independently of each other and can move either forward (positive speed) or backward (negative speed). Since the wheels do not pivot, to turn left the left wheels must be moving slower than the right wheels. Similarly to turn right the right wheels must be going slower than the left wheels. The goal is to create a system that will let the robot move forward, backward, and turn left or right while moving forward. Every time the direction of the robot is inputted by the user, the robot should print out the speed each wheel is going. The robot should also continuously check for user input until it is told to stop by the user. 
+
+![alt text](exercise_1.1_robot.PNG)
+
+# Starting Code
+We have already given you a main() function to write your code in and have included the proper libraries for you to use. We have also set up most of the while loop where you will write most of your code; the keep_going boolean will be used to trigger then end of the while loop, so you will need to change its value at some point. We have also set up the user input which is stored in the char variable, direction; this is the variable that you need to check for what was inputted.
+
+```c++
+//Do not edit the next 5 lines
+#include <stdlib.h>
+#include <iostream>
+#include <vector>
+
+int main(int argc, char** argv) {
+	std::cout << "Use the wasd keys and enter to input a direction, or the E key to stop" << std::endl;
+
+    // Write your code for step 1 here
+
+    // Write your code for step 2 here
+   
+    // Do not edit the next 5 lines
+    bool keep_going = true;
+    char direction;
```

Comment:
You fixed this in the solution, but the starter code and instructions need to be updated to initialize this variable.
```suggestion
    char direction = 0;
```